py/src/braintrust/logger.py

```diff
@@ -3810,6 +3810,10 @@ def set_current(self):
         if self.can_set_current:
             self._context_token = self.state.current_span.set(self)
 
+    def unset_current(self):
```

Comment:
I think we need to add this to the `Span` abstract class and all implementations. Not just `SpanImpl`.